Workspaces/Hierarchy/Prefabs/HierarchyListItem.prefab

```diff
@@ -216,7 +216,7 @@ MonoBehaviour:
   m_Name: 
   m_EditorClassIdentifier: 
   m_RunInEditMode: 0
-  m_SelectionFlags: 2
+  m_SelectionFlags: 3
```

Comment:
Consistency. Timoni (and Dennis) have asked that we switch everything to ray-and-direct "where possible." This is for the expand-arrow, which we weren't actually seeing until I changed its material to render on top of the list items.